### ISSUE TITLE
Поддержка комплексных компонент

### ISSUE THREAD

**Issue by Sypachev** [vizh/bxfs#1]
Здравствуйте, 
Протестировал решение, вначале был не прав, плагин работает, но основной функционал(переход на файлы компонентов) который требовался глючит.
Развернул локальную версию битрикса. На ней вроде по большей части все работает, правда не ищет default шаблоны
В разделе был компонент новостей, скачал шаблон компонента локально, но перейти нему не удалось
http://prnt.sc/e3u09t
Также не удалось перейти на простые компоненты в составе сложного http://prntscr.com/e3u3nx
phpstorm2016.1

**Comment by vizh** [comment]
Да, увы, но поддержка комплексных компонент ещё не была реализована. Только простых.
